### PR TITLE
nunify Connector

### PR DESCRIPTION
**Pull request by nunify-admin**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Hi @nunify-admin! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

File: certified-connectors/nunify/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+              "type": "object",
+              "properties": {
+                "email": {
+                  "type": "string",
+                  "description": "email",
+                  "title": "Email"
+                },
+                "first_name": {
+                  "type": "string",
+                  "description": "first_name",
```
Comment (sriyen-msft) [inline certified-connectors/nunify/apiDefinition.swagger.json:151]: @nunify-admin All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

File: certified-connectors/nunify/readme.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+## nunify
+
+nunify is an event tech platform to run any type of event - virtual events, hybrid events, webinars, in-person events & meetings. This platform provides all the tools for registrations, onsite check-in, networking, audience engagement, gamification, live stream, event apps, sponsorships and more. You can find out more about us at [nunify.com](https://www.nunify.com).
+
```
Comment (sriyen-msft) [inline certified-connectors/nunify/readme.md:4]: @nunify-admin Please add publisher

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by nunify-admin** [comment]
@sriyen-msft Thank you for your feedback. We have incorporated the same in the latest commit.

**Comment by nunify-admin** [comment]
@sriyen-msft We have implemented the requested changes. Could you please review and merge the same? Thanks!

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**